apps/pie-storybook/stories/testing/pie-thumbnail.test.stories.ts

```diff
@@ -50,28 +85,67 @@ const Template: TemplateFunction<ThumbnailProps> = ({
     variant,
     src,
     alt,
+    disabled,
+    hasPadding,
+    backgroundColor,
+    placeholder,
 }) => html`
     <pie-thumbnail
         variant="${variant}"
         src="${src}"
-        alt="${alt}">
+        alt="${alt}"
+        ?disabled="${disabled}"
+        ?hasPadding="${hasPadding}"
+        backgroundColor="${backgroundColor}"
+        .placeholder="${placeholder}">
     </pie-thumbnail>`;
 
 // Define the prop options for the matrix
-const sharedPropOptions = {
-    src: ['https://www.pie.design/assets/img/jet-logo-narrow.svg'],
-    alt: ['JET logo'],
+const sharedPropOptions: Partial<Record<keyof ThumbnailProps, unknown[]>> = {
+    src: [defaultArgs.src],
+    alt: [defaultArgs.alt],
+    disabled: [true, false],
+    hasPadding: [true, false],
+    backgroundColor: [defaultArgs.backgroundColor],
+    placeholder: [defaultArgs.placeholder],
 };
 
-const defaultPropOptions = {
+const defaultPropOptions: Partial<Record<keyof ThumbnailProps, unknown[]>> = {
     ...sharedPropOptions,
     variant: ['default'],
 };
 
-const outlinePropOptions = {
+const outlinePropOptions: Partial<Record<keyof ThumbnailProps, unknown[]>> = {
     ...sharedPropOptions,
     variant: ['outline'],
 };
 
+const backgroundPropOptions: Partial<Record<keyof ThumbnailProps, unknown[]>> = {
+    backgroundColor: backgroundColors,
+    variant: variants,
+    src: ['https://www.pie.design/assets/img/404_narrow.png'],
+};
+
+export const Default = createStory<ThumbnailProps>(Template, defaultArgs)();
+
+export const InvalidSrc = createStory<ThumbnailProps>(Template, {
+    placeholder: {
+        src: 'https://www.pie.design/assets/img/404_narrow.png',
+        alt: 'Placeholder Alt',
+    },
+    src: 'invalid-url.com',
+    alt: 'Invalid text',
+})();
+
+export const InvalidPlaceholder = createStory<ThumbnailProps>(Template, {
+    placeholder: {
+        src: 'https://www.pie.design/assets/img/404_narrow.png',
+        alt: 'Placeholder Alt',
+    },
+    src: 'https://www.pie.design/assets/img/jet-logo-narrow.svg',
+    alt: 'JET Logo',
+})();
```

Comment:
the storyname might be wrong. it mentions `InvalidPlaceholder` but all of the props are valid and all images will load as expected.